### ISSUE TITLE
NumberFormatException: For input string

### ISSUE THREAD

**Issue by TexeiraThiago** [TexeiraThiago/CricketChrips#1]
I would like to understand why, when I call correcCases() method and try to parse to Double the app crashes giving the stack trance of
 java.lang.NumberFormatException: For input string: $result

The way is done, the app works, but the output is a String and I want it to be a Double. 

Just to understand the concept for behind what's happening

**Comment by giuroperto** [comment]
Hey Thiago,
Not sure if I understood properly... but do you mean you want the correctCases to return a double instead of a String?
Like this:

`private fun correcCases(temperature: Double): Double {...}`

If that is the case, the `df.format(temperature)` returns a String, therefore you will need to turn it into a Double (toDouble()).
I'm just not sure if that's exactly what you meant because the log is mentioning the $result but that is the complete String to show in the tvResult, right?

**Comment by TexeiraThiago** [comment]
Resolved moving the result text to resources and using in the Main activity "string.format" and in the resource passing %.2f
Lesson learned 

Answering: I'm just not sure if that's exactly what you meant because the log is mentioning the $result, but that is the complete String to show in the tvResult, right?
I kinda make a mistake using "$result". What I meat by that was the result that was being shown in the stake trace. But thanks anyway, I did some other improvements in the last commit, moving all hardcoded text to resources.
